<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwfMp3Looper</class>
 <widget class="QWidget" name="SwfMp3Looper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a looped MP3 in a SWF</string>
  </property>
  <layout class="QHBoxLayout" name="verticalLayout_999">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_199">
     <item>
      <widget class="QListView" name="listView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="verticalLayout_799">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout1">
       <item>
        <widget class="QLabel" name="labelSourceFile">
         <property name="text">
          <string>Audio Source File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sourceFileLineEdit">
         <property name="placeholderText">
          <string>(Required)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectFileButton">
         <property name="text">
          <string>Select File...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelClassName">
         <property name="text">
          <string>ActionScript 3 Class Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="classNameLineEdit">
         <property name="placeholderText">
          <string>(Required)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Use Default Audio Settings</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="audioSettingsGroupBox">
       <property name="title">
        <string>Audio Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Codec:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="codecComboBox">
            <item>
             <property name="text">
              <string>MP3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw (16-bit)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Sample Rate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sampleRateComboBox">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>11025</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22050</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>44100</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Channels:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="channelsComboBox">
            <item>
             <property name="text">
              <string>Same as source</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stereo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mono</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="algorithmQualityGroup">
          <property name="title">
           <string>Algorithmic Quality</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSlider" name="algorithmicQualitySlider">
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>0 (Higher quality, slower encode)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>(Lower quality, faster encode) 9</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="vbrQualityGroup">
          <property name="title">
           <string>VBR Quality</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSlider" name="vbrQualitySlider">
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>0 (Higher quality, larger file)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>(Lower quality, smaller file) 9</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;a href=&quot;http://www.mjbshaw.com&quot; style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.mjbshaw.com&lt;/a&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save SWF As...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
